card/src/main/java/com/adyen/checkout/card/CardConfiguration.kt

```diff
@@ -216,6 +216,9 @@ class CardConfiguration private constructor(
         /**
          * Configures the installment options to be provided to the shopper.
          *
+         * When using `sessions` installment configuration will be ignored and replaced with the value
```

Comment:
Can you add a similar comment to `CardConfiguration.Builder.setShowStorePaymentField`?

Reply:
Same for bcmc, we missed this one